src/components/button/template.njk

```diff
@@ -10,9 +21,14 @@
   {% endif %}
 {% endif %}
 
+{% if params.isStartButton %}
```

Comment:
Based on our other API options like 'expanded' on the accordion, this should probably be something like `params.start`?

Reply:
I based this on fieldset's `isPageHeading` option. I also like `is` as it helps the user understand that the API is expecting a Boolean.  We also had a chat about this offline and both agreed this would be alright.